core/src/main/resources/hudson/model/Node/help-name.html

```diff
@@ -1,7 +1,9 @@
 <div>
-  Name that uniquely identifies a agent within this Jenkins installation.
-
+  Name that uniquely identifies an agent within this Jenkins installation.
   <p>
-  This value could be any string, and doesn't have to be the same as the agent host name,
-  but it's often convenient to make them the same.
+  This does not have to be the same as the agent hostname (where applicable),
+  but it is often convenient to make them the same.
+  <p>
+  The name may not contain any characters from this list:
+  <tt>?*/\\%!@#$^&|<>[]:;</tt>
```

Comment:
right. /me think this would still preferable in general as IMO it's a browser matter and we're not sure it's gonna render fine everywhere (or less sure I guess).